.github/workflows/python_lint.yml

```diff
@@ -0,0 +1,38 @@
+name: 💠 Python lint
+
+on:
+  push:
+    branches:
+      - current
+      - next
```

Comment:
What is the branching naming strategy here, out of curiosity?

Reply:
@theovald : Trying out new things. `current` for current release (main in conventional word), `next` for next release (dev in conventional word)